apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -579,22 +590,79 @@ public long countBlobs(String containerName, ListContainerOptions options) {
       throw new UnsupportedOperationException();
    }
 
-   // copied from BaseBlobStore
+   // TODO: Move statics up to BaseBlobStore to reuse?
+   @com.google.inject.Inject(optional = true)
+   @Named("jclouds.mpu.parallel.degree")
+   @VisibleForTesting
+   private int numThreads = 16;
+
+   private ExecutorService jcloudsBigBlobUploadTreadPool = MoreExecutors.getExitingExecutorService(
```

Comment:
Incredibly, none of the multipart configuration variables actually *do* anything, except for this one.  We should just remove all of them.